Annotato/Annotato/ViewModels/Document/DocumentAnnotationViewModel.swift

```diff
@@ -1,15 +1,32 @@
 import CoreGraphics
+import PDFKit
 
 class DocumentAnnotationViewModel {
-    private(set) var center: CGPoint
     private(set) var parts: [DocumentAnnotationPartViewModel]
     private(set) var width: Double
 
-    init(center: CGPoint, width: Double, parts: [DocumentAnnotationPartViewModel]) {
-        self.center = center
+    unowned var associatedDocumentPdfViewModel: DocumentPdfViewModel?
+    var associatedDocument: PDFDocument
```

Comment:
1. Why do we need to know this? I can't seem to find where it is used.
2. Why do we need to store this? Can't we just get it from `associatedDocumentPdfViewModel.associatedDocument` everytime we need it -- aka seems like it should be a derived property/ calculated properly. If we store it separately, suppose the `associatedDocumentPdfViewModel` changes, we may forget to update it.

Reply:
I added it in so that in the future we can persist the data
Shall I take it out first?

Ok the associated document I'll take it out